website/source/docs/import/importability.html.md

```diff
@@ -121,6 +121,7 @@ To make a resource importable, please see the
 * azurerm_sql_firewall_rule
 * azurerm_storage_account
 * azurerm_virtual_network
+* azurerm_image
```

Comment:
_minor_: these are currently sorted alphabetically for easier reading - could we re-order this? :)